tests/conftest.py

```diff
@@ -82,7 +82,7 @@ def mol_mol_regression_data(data_dir):
 
 @pytest.fixture
 def rxn_mol_regression_data(data_dir):
-    df = pd.read_csv(data_dir / "regression/rxn+mol.csv")
+    df = pd.read_csv(data_dir / "regression" / "rxn+mol" / "rxn+mol.csv")
```

Comment:
I see. The reason I suggested this is because the format isn't always consistent. If you look at the paths mentioned above this code block, you'll see they are using a different method.